src/hotspot/os/linux/globals_linux.hpp

```diff
@@ -77,7 +77,18 @@
           "Use CPU_ALLOC code path in os::active_processor_count ")     \
                                                                         \
   product(bool, DumpPerfMapAtExit, false, DIAGNOSTIC,                   \
-          "Write map file for Linux perf tool at exit")
+          "Write map file for Linux perf tool at exit")                 \
+                                                                        \
+  product(intx, TimerSlack, -1, EXPERIMENTAL,                           \
```

Comment:
Why experimental? Is this not something that these users want to use in production?

Reply:
I would say it is an option that can be used sparingly in selected production/experimental workloads. Like other low-level VM options, it stays experimental to avoid a stronger contract for supportability. I can turn that bona fide `product`, if you want.

Reply:
What do your "customers" want? Are they happy to deploy with an experimental flag on the command-line?

Reply:
Yes, experimental flag is fine for limited internal use. We can promote this flag to product if/when it proves to be useful for the generic low-latency cases.